modules/nw-anp-nodes-peer-concept.adoc

```diff
@@ -0,0 +1,243 @@
+
+In addition to supporting east-west traffic controls, ANP and BANP also allow administrators to control their northbound traffic leaving the cluster or traffic leaving the node to other nodes in the cluster. End-users can do the following:
+
+* Implement egress traffic control towards cluster nodes using `nodes` egress peer
+
+* Implement egress traffic control towards Kubernetes API servers using `nodes` or `networks` egress peers
+
+* Implement egress traffic control towards external destinations outside the cluster using `networks` peer
+
+[id="egress-traffic-control-towards-cluster-nodes_{context}"]
+== Using nodes peer to control egress traffic to cluster nodes
+
+Using the `nodes` peer administrators can control egress traffic from pods to nodes in the cluster.
+
+The following example allows egress traffic to the Kubernetes API server on port `6443` by any of the namespaces with a `restricted`, `confidential`, or `internal` level of security and in the `dept: engr` tenancy.
+
+.Example of ANP `Allow` egress using `nodes` peer
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: egress-security-allow
+spec:
+  egress:
+  - action: Deny
+    to:
+    - nodes:
+        matchExpressions:
+        - key: node-role.kubernetes.io/worker
+          operator: Exists
+  - action: Allow
+    name: allow-to-kubernetes-api-server
+    ports:
+    - portNumber:
+        port: 6443
+        protocol: TCP
+    - portNumber:
+        port: 443
+        protocol: TCP
+    to:
+    - nodes:
+        matchExpressions:
+        - key: node-role.kubernetes.io/control-plane <1>
+          operator: Exists
+    - pods:
+        namespaceSelector:
+          matchLabels:
+            dept: engr <2>
+        podSelector: {}
+  priority: 55
+  subject: <3>
+    namespaces:
+      matchExpressions:
+      - key: security
+        operator: In
+        values: <4>
+        - restricted
+        - confidential
+        - internal
+----
+<1> Specifies a node or set of nodes in the cluster using the `matchExpressions` field. It will select all nodes if the label is present, but left empty.
+<2> Specifies all the pods labeled with `dept: engr`.
+<3> Specifies the subject of the ANP which includes any namespaces that match the labels used by the network policy. The example matches any of the namespaces with `restricted`, `confidential`, or `internal` level of security.
+<4> Specifies key/value pairs for `matchExpressions` field.
+====
+
+[id="egress-traffic-control-networks-peer-external-destinations_{context}"]
+== Using networks peer to control egress traffic towards external destinations
+
+Cluster administrators can also use CIDR ranges for `networks` peer and apply an egress traffic policy to those nodes. The following are benefits to using `nodes` and `networks` peer when blocking access from cluster workloads to specific ports on cluster nodes:
```

Comment:
IMO The content from The following are benefits.. and three bullet items should be on the top as they are applicable to both nodes and networks peers not specific to network peers. After line number 16. 

Last two bullets are for nodes peers

Reply:
Alternatively move them under last section of using nodes and networks both.

Reply:
I moved the one up to the bullet points where you wanted it. The other I used as a framing sentence for this section and the third one I dropped because there was outstanding debate on it.